invenio_app_ils/config.py

```diff
@@ -113,8 +113,6 @@ def _(x):
 ###############################################################################
 #: Email address for support.
 SUPPORT_EMAIL = "info@inveniosoftware.org"
-#: Management email for internal notifications.
-MANAGEMENT_EMAIL = "internal@inveniosoftware.org"
```

Comment:
there were dependencies of this config var if I'm not wrong, but I don't see you removing any?

Reply:
dead code, not used any more because the email has been moved to CDS-ILS.